### ISSUE TITLE
floonet was reported with red disconnected icon while node info was available

### ISSUE THREAD

**Issue by cgilliard** [mwcproject/mwc-qt-wallet#86]
I seemed to be connected, but the icon was red. See screenshot.
<img width="917" alt="Screen Shot 2019-09-01 at 11 51 09 PM" src="https://user-images.githubusercontent.com/7232183/64095399-8de24600-cd13-11e9-803f-b99f96b5dfa6.png">

**Comment by cgilliard** [comment]
Also, clicking start said "listener is already started",but could not click "stop"
<img width="861" alt="Screen Shot 2019-09-01 at 11 52 54 PM" src="https://user-images.githubusercontent.com/7232183/64095458-af433200-cd13-11e9-9af9-04c0af78a5c3.png">

**Comment by bayk** [comment]
Chris, probably that indicator is confusing.  It doesn't report about the node. It is listener status.
Listener said 'it is already started' because it is started but it is offline. It is trying to reconnect in background.
Let's talk on discord about indicator. I think we need two indicators, one for node, another for listeners

**Comment by cgilliard** [comment]
Sure, we can talk about it. I'm traveling today, but will try to get into
discord later. That indicator is always red for me now.

**Comment by bayk** [comment]
We discuss that problem with Brent, please check discord for details.

**Comment by cgilliard** [comment]
I saw the discussion and responded. I think what you and brent propose is great, but I think there are two issues:

1) Confusion about what the red/green icon indicates. The stuff you and Brent discussed will address this issue.
2) Bug where icon is always red. I seem to be able to reproduce this issue by opening an mwc713 instance at the same time as the wallet and starting the listeners in mwc713. That causes the icon to stay red forever. The only way I was able to get it green again was to re-install the wallet. Even a restart did not help. If that's the only way it can happen it's probably a minor issue, but I'm concerned that it gets stuck in that state no matter what. Might be some other ways to trigger this state. Should investigate.

**Comment by bayk** [comment]
#2 is interesting, I never see that. Will try to reproduce

**Comment by cgilliard** [comment]
It might be related to the new disconnection/connection format. If you are running both the mwcmq and mwcmqs listener, here's the messages you will get on disconnect/reconnect:

wallet713> 
WARNING: mwcmqs listener [xmgehLCvsHXdoAenuUARvQGMPkzJdm5Qb1wTQVi2WnRaSjC1CWgH] lost connection. Will try to restore in the background.
WARNING: listener [xmgehLCvsHXdoAenuUARvQGMPkzJdm5Qb1wTQVi2WnRaSjC1CWgH] lost connection. it will keep trying to restore connection in the background.
INFO: listener [xmgehLCvsHXdoAenuUARvQGMPkzJdm5Qb1wTQVi2WnRaSjC1CWgH] reestablished connection.
wallet713> INFO: mwcmqs listener [xmgehLCvsHXdoAenuUARvQGMPkzJdm5Qb1wTQVi2WnRaSjC1CWgH] reestablished connection.

Basically they are the same except one of the messages says "mwcmqs" and the other doesn't (for both INFO and WARNING).

I'm thinking that we should drop the old version at this point since mwcmqs basically does everything we need and is getting stable.

**Comment by bayk** [comment]
We should support both. Most parsers already updated to support such
messages. Probably I miss those ones. I will update them.

**Comment by cgilliard** [comment]
The main thing I don't want to support is running the instance of grinbox.
It requires rabbitmq and I had to jump through a bunch of hoops to get it
running. I'm moving everything over to AWS because I can get better servers
for cheaper and I don't intend to set up grinbox. Here's what we can do:

I will make mwcmqs default on mwc713 (it will start automatically and it
will be the default address type if one is not specified), So if you run
listen or stop it will start mwcmqs. I'll make an option for grinbox, maybe
listen -g and stop -g. But we won't have a server that we support for
users, they will need to setup their own.

**Comment by bayk** [comment]
mwc713 already has options for grinbox and I am using that. No needs for
' -g'.  Enough to change default for listen and stop.
For qt wallet let's keep them. By default it is mqs.  I can clean up
default config file, so it will be not clear hoe to switch.  With that qt
wallet will always use mqs.
Let's keep support for MQ for some time in case of emergency. You don't
have to run MQ server for that. I would clean up the code after few months.


wallet713> help listen
listen
listens to incoming slates to your mwcmq address, mwcmqs address or keybase
account

USAGE:
    listen [FLAGS]

FLAGS:
    *-m, --mwcmq      start the mwcmq listener*
    -h, --help       Prints help information
    -k, --keybase    start the keybase listener
   * -s, --mwcmqs     start the mwcmqs listener*
    -V, --version    Prints version information

**Comment by cgilliard** [comment]
Yes, per discussion in discord, will keep it -m for mwcmq. -s will be
mwcmws, default will be -s (mwcmqs). We will continue to support both mwcmq
and mwcmqs in the qt wallet and mwc713, but will not run a grinbox server.
We will only run the mwcmqs server for users. Users that want to use
grinbox/mwcmq will need to setup their own instance of grinbox.

**Comment by bayk** [comment]
Chris, I created an issue about mwc mqs disconnect reporting time.
Message "listener for wmcmq allready stared" I will change to more clear 
"MWC MQS listener is running, but it lost connection and trying to reconnect in background to <mwc mq host name>. Please check your network connection"

**Comment by cgilliard** [comment]
The messages look similar mwcmqs is:

mwcmqs listener started for [
xmgehLCvsHXdoAenuUARvQGMPkzJdm5Qb1wTQVi2WnRaSjC1CWgH]

wallet713> exit


mwcmqs listener [xmgehLCvsHXdoAenuUARvQGMPkzJdm5Qb1wTQVi2WnRaSjC1CWgH]
stopped



and mwcmq is:


listener started for [xmgehLCvsHXdoAenuUARvQGMPkzJdm5Qb1wTQVi2WnRaSjC1CWgH]

wallet713> exit


listener [xmgehLCvsHXdoAenuUARvQGMPkzJdm5Qb1wTQVi2WnRaSjC1CWgH] stopped


Is that the confusion?

**Comment by bayk** [comment]
No, I already update that. Confusion is UI related.  At listener page because of connection, MWC MQ has red indicator.  Button is "Start". When you are pressing that button the error will be  'listener for wmcmq already stared'.  Message is confusing because of button "Start".
That what I fix. Just a message, mwc713 is fine. Will push soon.
